android/app/build.gradle

```diff
@@ -48,7 +48,7 @@ android {
         applicationId "fi.metatavu.noheva_visitor_ui"
         // You can update the following values to match your application needs.
         // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-gradle-build-configuration.
-        minSdkVersion flutter.minSdkVersion
+        minSdkVersion 19
```

Comment:
is this set to 19 on purpose? SDK 19 is really really old (see https://developer.android.com/tools/releases/platforms)

Reply:
`flutter.minSdkVersion` is even lower. It has been set due to a particular dependency (`device_info_plus` if I remember correctly) requires minimum SDK version of 19.

Would it be best to be kept as low as possible, or to correspond with initial customers Android device specifications?

Reply:
@Jdallos bump to 28